build.gradle

```diff
@@ -3,31 +3,101 @@ plugins {
     id 'java'
 }
 
-group 'com.fiskaly'
-version '0.0.1-alpha'
-
 subprojects {
     apply plugin: 'java'
+    apply plugin: 'maven'
+    apply plugin: 'signing'
+
+    group 'com.fiskaly.kassensichv'
```

Comment:
I'd say `com.fiskaly` should be our group namespace and `kassensichv` the artifact's prefix